### ISSUE TITLE
Dart Migrate Issue

### ISSUE THREAD

**Issue by akashgk** [flutter/flutter#84862]
## Steps to Reproduce

Ran the Command on the project
dart migrate --skip-import-check

Ran the Command on the project
dart migrate --skip-import-check

## Logs

Aborting migration due to an exception.  This most likely is due to a
bug in the migration tool.  Please consider filing a bug report at:

https://github.com/dart-lang/sdk/issues/new
Please include the SDK version (2.13.3) in your bug report.

To attempt to perform migration anyway, you may re-run with
--ignore-exceptions.

Exception details:

LateInitializationError: Field '_definingCompilationUnit@681486196' has not been initialized.
#0      LibraryElementImpl.source (package:analyzer/src/dart/element/element.dart:5248:12)
#1      MigrationResolutionHooksImpl.isLibraryNonNullableByDefault (package:nnbd_migration/src/fix_builder.dart:485:55)
#2      LibraryElementImpl.isNonNullableByDefault (package:analyzer/src/dart/element/element.dart:5125:35)
#3      LinkedElementFactory._setLibraryTypeSystem (package:analyzer/src/summary2/linked_element_factory.dart:524:40)
#4      LinkedElementFactory.createLibraryElementForReading (package:analyzer/src/summary2/linked_element_factory.dart:187:5)
#5      LinkedElementFactory.elementOfReference (package:analyzer/src/summary2/linked_element_factory.dart:258:14)
#6      LinkedElementFactory.elementOfReference (package:analyzer/src/summary2/linked_element_factory.dart:262:25)
#7      LinkedElementFactory.elementOfReference (package:analyzer/src/summary2/linked_element_factory.dart:262:25)
#8      LinkedResolutionReader._readRawElement (package:analyzer/src/summary2/bundle_reader.dart:820:27)
#9      LinkedResolutionReader.nextElement (package:analyzer/src/summary2/bundle_reader.dart:582:19)
#10     ApplyResolutionVisitor._nextElement (package:analyzer/src/summary2/apply_resolution.dart:1487:24)
#11     ApplyResolutionVisitor.visitSimpleIdentifier (package:analyzer/src/summary2/apply_resolution.dart:1172:26)
#12     SimpleIdentifierImpl.accept (package:analyzer/src/dart/ast/ast.dart:8836:50)
#13     ApplyResolutionVisitor.visitTypeName (package:analyzer/src/summary2/apply_resolution.dart:1258:15)
#14     TypeNameImpl.accept (package:analyzer/src/dart/ast/ast.dart:10068:50)
#15     ApplyResolutionVisitor.visitExtendsClause (package:analyzer/src/summary2/apply_resolution.dart:422:21)
#16     ExtendsClauseImpl.accept (package:analyzer/src/dart/ast/ast.dart:3642:50)
#17     ApplyResolutionVisitor.visitClassDeclaration (package:analyzer/src/summary2/apply_resolution.dart:197:27)
#18     ClassDeclarationImpl.accept (package:analyzer/src/dart/ast/ast.dart:1514:50)
#19     LinkedContext.applyResolution (package:analyzer/src/summary2/bundle_reader.dart:498:11)
#20     LinkedUnitContext.applyResolution (package:analyzer/src/summary2/linked_unit_context.dart:93:25)
#21     TypeParameterizedElementMixin.typeParameters (package:analyzer/src/dart/element/element.dart:7375:22)
#22     AlreadyMigratedCodeDecorator.decorate (package:nnbd_migration/src/already_migrated_code_decorator.dart:82:41)
#23     Variables._createDecoratedElementType (package:nnbd_migration/src/variables.dart:407:53)
#24     Variables.decoratedElementType (package:nnbd_migration/src/variables.dart:100:9)
#25     FixBuilder._computeMigratedType (package:nnbd_migration/src/fix_builder.dart:242:35)
#26     MigrationResolutionHooksImpl.getExecutableType.<anonymous closure> (package:nnbd_migration/src/fix_builder.dart:400:32)
#27     MigrationResolutionHooksImpl._wrapExceptions (package:nnbd_migration/src/fix_builder.dart:826:21)
#28     MigrationResolutionHooksImpl.getExecutableType (package:nnbd_migration/src/fix_builder.dart:399:7)
#29     MigrationResolutionHooksImpl.getExecutableParameters (package:nnbd_migration/src/fix_builder.dart:389:14)
#30     ExecutableElementImpl.parameters (package:analyzer/src/dart/element/element.dart:3579:35)
#31     CorrectOverrideHelper._computeThisTypeForSubtype (package:analyzer/src/error/correct_override.dart:72:34)
#32     new CorrectOverrideHelper (package:analyzer/src/error/correct_override.dart:32:5)
#33     InheritanceManager3.combineSignatures (package:analyzer/src/dart/element/inheritance_manager3.dart:82:28)
#34     InheritanceManager3._findMostSpecificFromNamedCandidates (package:analyzer/src/dart/element/inheritance_manager3.dart:446:31)
#35     InheritanceManager3._getInterfaceClass (package:analyzer/src/dart/element/inheritance_manager3.dart:559:9)
#36     InheritanceManager3.getInterface (package:analyzer/src/dart/element/inheritance_manager3.dart:219:18)
#37     InheritanceManager3.getMember2 (package:analyzer/src/dart/element/inheritance_manager3.dart:270:21)
#38     InheritanceManager3.getMember (package:analyzer/src/dart/element/inheritance_manager3.dart:237:22)
#39     TypePropertyResolver._lookupInterfaceType (package:analyzer/src/dart/resolver/type_property_resolver.dart:215:31)
#40     TypePropertyResolver.resolve (package:analyzer/src/dart/resolver/type_property_resolver.dart:160:9)
#41     PropertyElementResolver._resolve (package:analyzer/src/dart/resolver/property_element_resolver.dart:368:49)
#42     PropertyElementResolver.resolvePropertyAccess (package:analyzer/src/dart/resolver/property_element_resolver.dart:183:12)
#43     ResolverVisitor.visitPropertyAccess (package:analyzer/src/generated/resolver.dart:1919:27)
#44     PropertyAccessImpl.accept (package:analyzer/src/dart/ast/ast.dart:8314:50)
#45     ResolverVisitor.visitPropertyAccess (package:analyzer/src/generated/resolver.dart:1915:18)
#46     PropertyAccessImpl.accept (package:analyzer/src/dart/ast/ast.dart:8314:50)
#47     ResolverVisitor.visitMethodInvocation (package:analyzer/src/generated/resolver.dart:1811:13)
#48     MethodInvocationImpl.accept (package:analyzer/src/dart/ast/ast.dart:7091:50)
#49     VariableDeclarationResolver.resolve (package:analyzer/src/dart/resolver/variable_declaration_resolver.dart:54:17)
#50     ResolverVisitor.visitVariableDeclaration (package:analyzer/src/generated/resolver.dart:2156:34)
#51     VariableDeclarationImpl.accept (package:analyzer/src/dart/ast/ast.dart:10412:50)
#52     NodeListImpl.accept (package:analyzer/src/dart/ast/ast.dart:7517:20)
#53     VariableDeclarationListImpl.visitChildren (package:analyzer/src/dart/ast/ast.dart:10513:16)
#54     ResolverVisitor.visitNode (package:analyzer/src/generated/resolver.dart:1880:10)
#55     UnifyingAstVisitor.visitVariableDeclarationList (package:analyzer/dart/ast/visitor.dart:3493:7)
#56     ResolverVisitor.visitVariableDeclarationList (package:analyzer/src/generated/resolver.dart:2188:11)
#57     VariableDeclarationListImpl.accept (package:analyzer/src/dart/ast/ast.dart:10507:15)
#58     VariableDeclarationStatementImpl.visitChildren (package:analyzer/src/dart/ast/ast.dart:10560:19)
#59     ResolverVisitor.visitNode (package:analyzer/src/generated/resolver.dart:1880:10)
#60     UnifyingAstVisitor.visitVariableDeclarationStatement (package:analyzer/dart/ast/visitor.dart:3497:7)
#61     VariableDeclarationStatementImpl.accept (package:analyzer/src/dart/ast/ast.dart:10556:15)
#62     NodeListImpl.accept (package:analyzer/src/dart/ast/ast.dart:7517:20)
#63     BlockImpl.visitChildren (package:analyzer/src/dart/ast/ast.dart:1069:17)
#64     ResolverVisitor.visitNode (package:analyzer/src/generated/resolver.dart:1880:10)
#65     UnifyingAstVisitor.visitBlock (package:analyzer/dart/ast/visitor.dart:3147:32)
#66     ScopedVisitor.visitBlock.<anonymous closure> (package:analyzer/src/generated/resolver.dart:2624:13)
#67     ScopedVisitor._withDeclaredLocals (package:analyzer/src/generated/resolver.dart:3220:8)
#68     ScopedVisitor.visitBlock (package:analyzer/src/generated/resolver.dart:2623:5)
#69     BlockImpl.accept (package:analyzer/src/dart/ast/ast.dart:1065:50)
#70     BlockFunctionBodyImpl.visitChildren (package:analyzer/src/dart/ast/ast.dart:1024:12)
#71     ResolverVisitor.visitNode (package:analyzer/src/generated/resolver.dart:1880:10)
#72     UnifyingAstVisitor.visitBlockFunctionBody (package:analyzer/dart/ast/visitor.dart:3150:56)
#73     ScopedVisitor.visitBlockFunctionBody (package:analyzer/src/generated/resolver.dart:2633:13)
#74     ResolverVisitor.visitBlockFunctionBody (package:analyzer/src/generated/resolver.dart:1050:13)
#75     BlockFunctionBodyImpl.accept (package:analyzer/src/dart/ast/ast.dart:1020:50)
#76     MethodDeclarationImpl.visitChildren (package:analyzer/src/dart/ast/ast.dart:6951:11)
#77     ResolverVisitor.visitNode (package:analyzer/src/generated/resolver.dart:1880:10)
#78     UnifyingAstVisitor.visitMethodDeclaration (package:analyzer/dart/ast/visitor.dart:3359:56)
#79     ScopedVisitor.visitMethodDeclarationInScope (package:analyzer/src/generated/resolver.dart:3070:11)
#80     ScopedVisitor.visitMethodDeclaration (package:analyzer/src/generated/resolver.dart:3063:7)
#81     ResolverVisitor.visitMethodDeclaration (package:analyzer/src/generated/resolver.dart:1787:11)
#82     MethodDeclarationImpl.accept (package:analyzer/src/dart/ast/ast.dart:6942:50)
#83     NodeListImpl.accept (package:analyzer/src/dart/ast/ast.dart:7517:20)
#84     ScopedVisitor.visitClassMembersInScope (package:analyzer/src/generated/resolver.dart:2694:18)
#85     ScopedVisitor.visitClassDeclaration (package:analyzer/src/generated/resolver.dart:2675:7)
#86     ResolverVisitor.visitClassDeclaration (package:analyzer/src/generated/resolver.dart:1108:13)
#87     ClassDeclarationImpl.accept (package:analyzer/src/dart/ast/ast.dart:1514:50)
#88     ResolverVisitor.visitCompilationUnit (package:analyzer/src/generated/resolver.dart:1184:23)
#89     CompilationUnitImpl.accept (package:analyzer/src/dart/ast/ast.dart:2105:50)
#90     FixBuilder.visitAll (package:nnbd_migration/src/fix_builder.dart:210:12)
#91     NullabilityMigrationImpl.finalizeInput (package:nnbd_migration/src/nullability_migration_impl.dart:169:18)
#92     NonNullableFix.finalizeUnit (package:nnbd_migration/src/front_end/non_nullable_fix.dart:115:15)
#93     _FixCodeProcessor.runLaterPhases.<anonymous closure> (package:nnbd_migration/migration_cli.dart:1157:21)
#94     _FixCodeProcessor.runLaterPhases.<anonymous closure> (package:nnbd_migration/migration_cli.dart:1154:28)
#95     _FixCodeProcessor.processResources (package:nnbd_migration/migration_cli.dart:1079:30)
<asynchronous suspension>
#96     _FixCodeProcessor.runLaterPhases (package:nnbd_migration/migration_cli.dart:1154:5)
<asynchronous suspension>
#97     MigrationCliRunner.run (package:nnbd_migration/migration_cli.dart:747:24)
<asynchronous suspension>
#98     MigrateCommand.run (package:nnbd_migration/migration_cli.dart:184:7)
<asynchronous suspension>
#99     CommandRunner.runCommand (package:args/command_runner.dart:196:13)
<asynchronous suspension>
#100    DartdevRunner.runCommand (package:dartdev/dartdev.dart:199:18)
<asynchronous suspension>
#101    runDartdev (package:dartdev/dartdev.dart:53:16)
<asynchronous suspension>
#102    main (file:///opt/s/w/ir/cache/builder/src/third_party/dart/pkg/dartdev/bin/dartdev.dart:11:3)
<asynchronous suspension>

 at offset 0 in /Users/akashgkrishnan/Documents/ftl-web/ats-mobile/lib/models/email.dart (import 'package:html/parser.dart'; import 'package:intl/intl.dart'; import '../utils/helper.dart'; class Email {final int id; final int jobId; final int candidateId; final String threadId; final String subject; final String body; final String bodyInHTML; final String toEmail; final String toName; final String fromName; final String fromEmail; final List<String> attachments; final String createdAt; final DateTime date; final String recruiterImageUrl; bool isRead; final bool isSentEmail; bool isStarred; Email(this.id, {this.jobId, this.candidateId, this.threadId, this.subject, this.body, this.bodyInHTML, this.toEmail, this.toName, this.fromName, this.fromEmail, this.attachments, this.createdAt, this.date, this.recruiterImageUrl, this.isStarred, this.isSentEmail = false, this.isRead = false}); static DateTime parseDate(dynamic date) {DateTime createdOn; try {createdOn = DateTime.parse(date).toLocal();} catch (e) {} return createdOn;} static String parseDateToString(dynamic date) {String createdAt; try {createdAt = DateFormat.yMMMd().add_jm().format(DateTime.parse(date).toLocal()); return createdAt;} catch (e) {} return createdAt;} static String parseHtml(String string) {String parsedHtmlText = parse(string).documentElement.text.trim(); return parsedHtmlText;} static List<String> parseStringList(var list) => list == null ? null : List.from(list); static Email fromJson(var email) {return Email(email['id'], jobId: email['job'], candidateId: email['job_candidate_id'], threadId: email['thread_id'], fromName: email['from_name'], fromEmail: email['from_email'], subject: email['subject'], body: parseHtml(email['body']), bodyInHTML: utf8decode(email['body']), toName: email['to_name'], toEmail: email['to_email'], isRead: email['is_read'], createdAt: parseDateToString(email['created_at']), date: parseDate(email['created_at']), recruiterImageUrl: email['image_url'], attachments: parseStringList(email['attachments']), isStarred: email['is_starred']);} static List<Email> fromJsonList(List jsonList) {List<Email> emails = jsonList.map((email) => fromJson(email)).toList(); emails.sort((Email e2, Email e1) => e1?.date?.compareTo(e2?.date) ?? 0); return emails;}})

#0      ApplyResolutionVisitor.visitClassDeclaration (package:analyzer/src/summary2/apply_resolution.dart:210:7)
#1      ClassDeclarationImpl.accept (package:analyzer/src/dart/ast/ast.dart:1514:50)
#2      LinkedContext.applyResolution (package:analyzer/src/summary2/bundle_reader.dart:498:11)
#3      LinkedUnitContext.applyResolution (package:analyzer/src/summary2/linked_unit_context.dart:93:25)
#4      TypeParameterizedElementMixin.typeParameters (package:analyzer/src/dart/element/element.dart:7375:22)
#5      AlreadyMigratedCodeDecorator.decorate (package:nnbd_migration/src/already_migrated_code_decorator.dart:82:41)
#6      Variables._createDecoratedElementType (package:nnbd_migration/src/variables.dart:407:53)
#7      Variables.decoratedElementType (package:nnbd_migration/src/variables.dart:100:9)
#8      FixBuilder._computeMigratedType (package:nnbd_migration/src/fix_builder.dart:242:35)
#9      MigrationResolutionHooksImpl.getExecutableType.<anonymous closure> (package:nnbd_migration/src/fix_builder.dart:400:32)
#10     MigrationResolutionHooksImpl._wrapExceptions (package:nnbd_migration/src/fix_builder.dart:826:21)
#11     MigrationResolutionHooksImpl.getExecutableType (package:nnbd_migration/src/fix_builder.dart:399:7)
#12     MigrationResolutionHooksImpl.getExecutableParameters (package:nnbd_migration/src/fix_builder.dart:389:14)
#13     ExecutableElementImpl.parameters (package:analyzer/src/dart/element/element.dart:3579:35)
#14     CorrectOverrideHelper._computeThisTypeForSubtype (package:analyzer/src/error/correct_override.dart:72:34)
#15     new CorrectOverrideHelper (package:analyzer/src/error/correct_override.dart:32:5)
#16     InheritanceManager3.combineSignatures (package:analyzer/src/dart/element/inheritance_manager3.dart:82:28)
#17     InheritanceManager3._findMostSpecificFromNamedCandidates (package:analyzer/src/dart/element/inheritance_manager3.dart:446:31)
#18     InheritanceManager3._getInterfaceClass (package:analyzer/src/dart/element/inheritance_manager3.dart:559:9)
#19     InheritanceManager3.getInterface (package:analyzer/src/dart/element/inheritance_manager3.dart:219:18)
#20     InheritanceManager3.getMember2 (package:analyzer/src/dart/element/inheritance_manager3.dart:270:21)
#21     InheritanceManager3.getMember (package:analyzer/src/dart/element/inheritance_manager3.dart:237:22)
#22     TypePropertyResolver._lookupInterfaceType (package:analyzer/src/dart/resolver/type_property_resolver.dart:215:31)
#23     TypePropertyResolver.resolve (package:analyzer/src/dart/resolver/type_property_resolver.dart:160:9)
#24     PropertyElementResolver._resolve (package:analyzer/src/dart/resolver/property_element_resolver.dart:368:49)
#25     PropertyElementResolver.resolvePropertyAccess (package:analyzer/src/dart/resolver/property_element_resolver.dart:183:12)
#26     ResolverVisitor.visitPropertyAccess (package:analyzer/src/generated/resolver.dart:1919:27)
#27     PropertyAccessImpl.accept (package:analyzer/src/dart/ast/ast.dart:8314:50)
#28     ResolverVisitor.visitPropertyAccess (package:analyzer/src/generated/resolver.dart:1915:18)
#29     PropertyAccessImpl.accept (package:analyzer/src/dart/ast/ast.dart:8314:50)
#30     ResolverVisitor.visitMethodInvocation (package:analyzer/src/generated/resolver.dart:1811:13)
#31     MethodInvocationImpl.accept (package:analyzer/src/dart/ast/ast.dart:7091:50)
#32     VariableDeclarationResolver.resolve (package:analyzer/src/dart/resolver/variable_declaration_resolver.dart:54:17)
#33     ResolverVisitor.visitVariableDeclaration (package:analyzer/src/generated/resolver.dart:2156:34)
#34     VariableDeclarationImpl.accept (package:analyzer/src/dart/ast/ast.dart:10412:50)
#35     NodeListImpl.accept (package:analyzer/src/dart/ast/ast.dart:7517:20)
#36     VariableDeclarationListImpl.visitChildren (package:analyzer/src/dart/ast/ast.dart:10513:16)
#37     ResolverVisitor.visitNode (package:analyzer/src/generated/resolver.dart:1880:10)
#38     UnifyingAstVisitor.visitVariableDeclarationList (package:analyzer/dart/ast/visitor.dart:3493:7)
#39     ResolverVisitor.visitVariableDeclarationList (package:analyzer/src/generated/resolver.dart:2188:11)
#40     VariableDeclarationListImpl.accept (package:analyzer/src/dart/ast/ast.dart:10507:15)
#41     VariableDeclarationStatementImpl.visitChildren (package:analyzer/src/dart/ast/ast.dart:10560:19)
#42     ResolverVisitor.visitNode (package:analyzer/src/generated/resolver.dart:1880:10)
#43     UnifyingAstVisitor.visitVariableDeclarationStatement (package:analyzer/dart/ast/visitor.dart:3497:7)
#44     VariableDeclarationStatementImpl.accept (package:analyzer/src/dart/ast/ast.dart:10556:15)
#45     NodeListImpl.accept (package:analyzer/src/dart/ast/ast.dart:7517:20)
#46     BlockImpl.visitChildren (package:analyzer/src/dart/ast/ast.dart:1069:17)
#47     ResolverVisitor.visitNode (package:analyzer/src/generated/resolver.dart:1880:10)
#48     UnifyingAstVisitor.visitBlock (package:analyzer/dart/ast/visitor.dart:3147:32)
#49     ScopedVisitor.visitBlock.<anonymous closure> (package:analyzer/src/generated/resolver.dart:2624:13)
#50     ScopedVisitor._withDeclaredLocals (package:analyzer/src/generated/resolver.dart:3220:8)
#51     ScopedVisitor.visitBlock (package:analyzer/src/generated/resolver.dart:2623:5)
#52     BlockImpl.accept (package:analyzer/src/dart/ast/ast.dart:1065:50)
#53     BlockFunctionBodyImpl.visitChildren (package:analyzer/src/dart/ast/ast.dart:1024:12)
#54     ResolverVisitor.visitNode (package:analyzer/src/generated/resolver.dart:1880:10)
#55     UnifyingAstVisitor.visitBlockFunctionBody (package:analyzer/dart/ast/visitor.dart:3150:56)
#56     ScopedVisitor.visitBlockFunctionBody (package:analyzer/src/generated/resolver.dart:2633:13)
#57     ResolverVisitor.visitBlockFunctionBody (package:analyzer/src/generated/resolver.dart:1050:13)
#58     BlockFunctionBodyImpl.accept (package:analyzer/src/dart/ast/ast.dart:1020:50)
#59     MethodDeclarationImpl.visitChildren (package:analyzer/src/dart/ast/ast.dart:6951:11)
#60     ResolverVisitor.visitNode (package:analyzer/src/generated/resolver.dart:1880:10)
#61     UnifyingAstVisitor.visitMethodDeclaration (package:analyzer/dart/ast/visitor.dart:3359:56)
#62     ScopedVisitor.visitMethodDeclarationInScope (package:analyzer/src/generated/resolver.dart:3070:11)
#63     ScopedVisitor.visitMethodDeclaration (package:analyzer/src/generated/resolver.dart:3063:7)
#64     ResolverVisitor.visitMethodDeclaration (package:analyzer/src/generated/resolver.dart:1787:11)
#65     MethodDeclarationImpl.accept (package:analyzer/src/dart/ast/ast.dart:6942:50)
#66     NodeListImpl.accept (package:analyzer/src/dart/ast/ast.dart:7517:20)
#67     ScopedVisitor.visitClassMembersInScope (package:analyzer/src/generated/resolver.dart:2694:18)
#68     ScopedVisitor.visitClassDeclaration (package:analyzer/src/generated/resolver.dart:2675:7)
#69     ResolverVisitor.visitClassDeclaration (package:analyzer/src/generated/resolver.dart:1108:13)
#70     ClassDeclarationImpl.accept (package:analyzer/src/dart/ast/ast.dart:1514:50)
#71     ResolverVisitor.visitCompilationUnit (package:analyzer/src/generated/resolver.dart:1184:23)
#72     CompilationUnitImpl.accept (package:analyzer/src/dart/ast/ast.dart:2105:50)
#73     FixBuilder.visitAll (package:nnbd_migration/src/fix_builder.dart:210:12)
#74     NullabilityMigrationImpl.finalizeInput (package:nnbd_migration/src/nullability_migration_impl.dart:169:18)
#75     NonNullableFix.finalizeUnit (package:nnbd_migration/src/front_end/non_nullable_fix.dart:115:15)
#76     _FixCodeProcessor.runLaterPhases.<anonymous closure> (package:nnbd_migration/migration_cli.dart:1157:21)
#77     _FixCodeProcessor.runLaterPhases.<anonymous closure> (package:nnbd_migration/migration_cli.dart:1154:28)
#78     _FixCodeProcessor.processResources (package:nnbd_migration/migration_cli.dart:1079:30)
<asynchronous suspension>
#79     _FixCodeProcessor.runLaterPhases (package:nnbd_migration/migration_cli.dart:1154:5)
<asynchronous suspension>
#80     MigrationCliRunner.run (package:nnbd_migration/migration_cli.dart:747:24)
<asynchronous suspension>
#81     MigrateCommand.run (package:nnbd_migration/migration_cli.dart:184:7)
<asynchronous suspension>
#82     CommandRunner.runCommand (package:args/command_runner.dart:196:13)
<asynchronous suspension>
#83     DartdevRunner.runCommand (package:dartdev/dartdev.dart:199:18)
<asynchronous suspension>
#84     runDartdev (package:dartdev/dartdev.dart:53:16)
<asynchronous suspension>
#85     main (file:///opt/s/w/ir/cache/builder/src/third_party/dart/pkg/dartdev/bin/dartdev.dart:11:3)
<asynchronous suspension>

<details>
<summary>Logs</summary>

[✓] Flutter (Channel stable, 2.2.2, on macOS 11.4 20F71 darwin-arm, locale en-IN)
    • Flutter version 2.2.2 at /Users/akashgkrishnan/Downloads/flutter
    • Framework revision d79295af24 (7 days ago), 2021-06-11 08:56:01 -0700
    • Engine revision 91c9fc8fe0
    • Dart version 2.13.3

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/akashgkrishnan/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.5, Build version 12E262
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6916264)

[✓] VS Code (version 1.57.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.23.0

[✓] Connected device (2 available)
    • iPhone 12 Pro Max (mobile) • DAD9B8E1-305F-489E-996F-B385CC034814 • ios            •
      com.apple.CoreSimulator.SimRuntime.iOS-14-5 (simulator)
    • Chrome (web)               • chrome                               • web-javascript • Google Chrome 91.0.4472.106

• No issues found!

</details>

**Comment by pengboboer** [comment]
I have the same mistake. What's the reason for this? I can't understand it

**Comment by stuartmorgan** [comment]
As it says in the output:

> Please consider filing a bug report at:
> 
> https://github.com/dart-lang/sdk/issues/new

`migrate` is part of Dart, not Flutter, so doesn't use this issue tracker.

**Comment by pengboboer** [comment]
I found a temporary solution. I found the problematic file in the error report and commented out the code of that file. Then try again. This problem also appeared in another file for a while, so continue to comment these codes. I tried four times and dart migrate was successful.

I don't know what is wrong with this, but a plug-in about the map is imported in these files with errors, which may be the reason for this plug-in.
